<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientMonitor</class>
 <widget class="QMainWindow" name="PatientMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PatientMonitor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(142, 157, 204);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="state">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>731</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../images/nav.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pulseSortButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
	border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 20px;
}

QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
    </property>
    <property name="text">
     <string>Rh group | Pulse sort</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9DBF1;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../images/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="rectangle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../images/menu.png</pixmap>
    </property>
   </widget>
   <widget class="QTableWidget" name="patientTable">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>711</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
	background-color: #D9DBF1;
    color: #000000;
    border-radius: 20px;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}

QTableWidget::item:hover { 
border-right: 0.5px solid #E5E5E5;
}

QTableWidget::item:hover { 
border-right: 0.5px solid #E5E5E5;
background-color: #A9B9E3
 }

QTableWidget::item:active { 
border-right: 0.5px solid #E5E5E5;
background-color: #A9B9E3
 }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writeButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>70</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}

QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bloodPressureSort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
	border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 20px;
}

QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
    </property>
    <property name="text">
     <string>Sort by pressure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="donorCheckButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
	border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 20px;
}

QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
    </property>
    <property name="text">
     <string>Check donors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>70</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}

QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scanButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>70</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}

QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agePulseCheckButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
	border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 20px;
}

QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
    </property>
    <property name="text">
     <string>Age pressure checker</string>
    </property>
   </widget>
   <widget class="QPushButton" name="normPressureCheckButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
	border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 20px;
}

QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}</string>
    </property>
    <property name="text">
     <string>Check healthy person</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bloodTypeGroupButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
	border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 20px;
}

QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
    </property>
    <property name="text">
     <string>Group by BT | RH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scanTableButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #C6CCE8;
color: #000000;
border-radius: 0.5em;
font-family: 'Gogh';
font-weight: 500;
font-size: 30px
}

QPushButton:hover {
background-color: #A9B9E3;
font-family: 'Gogh';
font-weight: 500;
font-size: 30px
}</string>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rhGroupHrSort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
	border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 20px;
}

QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
    </property>
    <property name="text">
     <string>Group RH | Sort HR</string>
    </property>
   </widget>
   <zorder>state</zorder>
   <zorder>pulseSortButton</zorder>
   <zorder>rectangle</zorder>
   <zorder>patientTable</zorder>
   <zorder>writeButton</zorder>
   <zorder>bloodPressureSort</zorder>
   <zorder>donorCheckButton</zorder>
   <zorder>addButton</zorder>
   <zorder>scanButton</zorder>
   <zorder>agePulseCheckButton</zorder>
   <zorder>normPressureCheckButton</zorder>
   <zorder>bloodTypeGroupButton</zorder>
   <zorder>logo</zorder>
   <zorder>scanTableButton</zorder>
   <zorder>rhGroupHrSort</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
